### ISSUE TITLE
Losing Shopping Lists/Full reset of Addon

### ISSUE THREAD

**Issue by Kareshkah** [Auctionator/Journalator#33]
Recently Ive been having issues with Auctionator Shopping lists resetting. I log-on to roughly 58 different servers per day and have had this reset happen multiple times, enough to where I have counted how many log-ins I have after a reset, seems to be on the 50th or 51st log in where I receive a lua error with the error below. Moments ago this reset happened again but without the lua error. I implemented the /atr c auctions_history_length 3 fix mentioned in Discord on every single character running Auctionator with no luck. 

1x FrameXML\Bindings.xml:1 FrameXML\Bindings.xml:1 constant table overflow
1x FrameXML\Bindings.xml:1 constant table overflow
1x constant table overflow

**Comment by plusmouse** [comment]
Do you use the price histories from Auctionator?

**Comment by Kareshkah** [comment]
I don't believe so..is that an additional addon/plugin that is downloaded with Auctionator?

**Comment by plusmouse** [comment]
If you don't use the price histories (tooltip values or selling tab listing) you can turn them off with ```
/atr nopricedb
```
That will resolve the reset issue

**Comment by Kareshkah** [comment]
Ahh I understand now, sorry I misunderstood what you were asking! I will run that right now, thank you. The only thing I use the Shopping lists for are PointBlankSniper and BoE snipes list.
